tests/atoms/gaia_apps.js

```diff
@@ -372,12 +372,31 @@ var GaiaApps = {
     };
     return result;
   },
+  /**
+  * Install the app with the specified ManifestURL
+  */
+  installWithManifestURL: function(manifestURL) {
```

Comment:
This function is a duplication of `uninstallWithManifestURL`. Duplicate will introduce many changes to perform, if one day the APIs change. In order to prevent that, how about:
```js

installWithManifestURL: function(manifestURL) {
    changeStateOfApp(manifestURL, navigator.mozApps.mgmt.install)
},

uninstallWithManifestURL: function(manifestURL) {
    changeStateOfApp(manifestURL, navigator.mozApps.mgmt.uninstall)
},

changeStateOfApp: function(manifestURL, functionToChangeState) {
  GaiaApps.locateWithManifestURL(manifestURL, function uninstall(app) {
    if (typeof(app) === 'object') {
      let req = functionToChangeState(app);
        req.onsuccess = function() {
        marionetteScriptFinished(true);
      };
      req.onerror = function() {
        marionetteScriptFinished(req.error);
      };
    } else {
      // App is already in this state, so nothing to do here
      marionetteScriptFinished(true);
    }
  });
}
```